README.md

```diff
@@ -72,11 +81,16 @@ cd ../..
 
 ### Install CPyCppyy
 
+Create virtual environment and activate it:
+```
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
 Clone the repo, checkout `rm-root-meta` branch and build it:
 ```
-git clone https://github.com/sudo-panda/CPyCppyy.git
+git clone https://github.com/compiler-research/CPyCppyy.git
```

Comment:
These are different: https://github.com/compiler-research/cppyy-backend/blob/abba84d9bece2742c02fd189a70063d4d527eb8e/.github/workflows/ci.yml#L170

Reply:
Why did the PR you were working on with alexander not get merged?

Reply:
I see the PR was merged but the setup is still commented out.

Reply:
Over here: https://github.com/compiler-research/cppyy-backend/actions/runs/4314140722/workflow

Reply:
But the master branch has it uncommented. That is weird.

Reply:
you need to rebase.

Reply:
Ok I need to rebase